woocommerce/woocommerce
# Add global coming soon property tracks

### Changes proposed in this Pull Request:

Closes https://github.com/woocommerce/woocommerce/issues/47642.

Adds `coming_soon` tracks property via `woocommerce_tracks_event_properties`


### How to test the changes in this Pull Request:

1. Set up a new store, make sure to **not skip** the core profiler so that tracking is enabled
1. Go to `WooCommerce > Settings > Site visibility`
4. Ensure `Coming soon` is selected and `Restrict to store pages only` is disabled
1. Go to `WooCommerce > Home`
2. Open up browser console and run `wcTracks.recordEvent('test_coming_soon', {})`
3. Go to networks tab and filter for `test_coming_soon`
4. Click on it and view the request payload
5. Observe `coming_soon: site` exists 
6. Go to `WooCommerce > Settings > Site visibility` and enable `Restrict to store pages only`
7. Refresh homescreen and run `wcTracks.recordEvent('test_coming_soon', {})`
5. Observe `coming_soon: store` exists 
6. Go to `WooCommerce > Settings > Site visibility` and set to `Live`
7. Refresh homescreen and run `wcTracks.recordEvent('test_coming_soon', {})`
5. Observe `coming_soon: no` exists 

<img width="832" alt="image" src="https://github.com/woocommerce/woocommerce/assets/3747241/54e40d8b-dcdb-49f3-84fc-d44f5c80f57c">

### Changelog entry




<details>

#### Significance




#### Type




#### Message 

#### Comment 

</details>
